### DISCUSSION TITLE
Setting Anchor points on images?

### DISCUSSION THREAD

**Discussion by defunky** [SixLabors/ImageSharp#2041] [Ideas] [answered]
Is it possible to set an Anchor point on the images when drawing it? I know it uses top-left when setting the point on when using .DrawImage() but is it possible to change this to be center for example?

**Comment by JimBobSquarePants** [comment, marked as answer]
No. You'll need to work that position yourself.

**Reply by agusehrman** [reply]
Im currently using drawImage to  place images on my PDF but it seems to be anchoring the coordinates on the bottom left corner of the image. How can I set the anchor as top-left ?

**Reply by tocsoft** [reply]
@agusehrman we have nothing to do with PDF generation so you need to take that up with what ever library your using for generating PDFs.
